### PR TITLE
Fix various issues mentioned in this weeks meeting.

### PR DESCRIPTION
**Pull request by MarkusFrankATcernch**

### CODE REVIEW

File: DDCore/include/DD4hep/objects/VolumeManagerInterna.h [hunk 1]
```diff
@@ -54,7 +54,7 @@ namespace DD4hep {
       /// Placement identifier
       VolumeID     identifier;
       /// Ignore mask of the placement identifier
-      [[gnu::deprecated("This member variable might get axed if it is not used, please tell us if you do")]]
+      //[[gnu::deprecated("This member variable might get axed if it is not used, please tell us if you do")]]
```
Comment (petricm) [inline DDCore/include/DD4hep/objects/VolumeManagerInterna.h]: Why is just commented out?

File: DDCore/src/plugins/VolumeMgrTest.cpp [hunk 2]
```diff
@@ -81,99 +84,177 @@ VolIDTest::VolIDTest(LCDD& lcdd, DetElement sdet, size_t depth) : m_det(sdet) {
...
+      //bool is_sensitive = place.volume().isSensitive();
+      //if ( is_sensitive || !child_ids.empty() )  {
+        checkVolume(detector, place, child_ids, child_chain);
+        //}
+      walkVolume(detector, place, child_ids, child_chain, depth+1, mx_depth);
     }
   }
 }
-
+#if 0
```
Comment (petricm) [inline DDCore/src/plugins/VolumeMgrTest.cpp]: Whya re you keeping this in the code? This was replaced with walkvolume?

File: DDCore/include/DD4hep/objects/VolumeManagerInterna.h (hunk 1 above)
Comment (andresailer) [inline DDCore/include/DD4hep/objects/VolumeManagerInterna.h]: Did you forget to uncomment this, or is the `mask` no longer being phased out?

File: DDCore/src/plugins/VolumeMgrTest.cpp (hunk 2 above)
Comment (MarkusFrankATcernch) [inline DDCore/src/plugins/VolumeMgrTest.cpp]: This test will be further elaborated in the future, but I do not yet know exactly what should go there. So let's keep it.

File: DDCore/include/DD4hep/objects/VolumeManagerInterna.h (hunk 1 above)
Comment (MarkusFrankATcernch) [inline DDCore/include/DD4hep/objects/VolumeManagerInterna.h]: This is intentional. The mask very likely cannot go away.

Comment (andresailer) [inline DDCore/include/DD4hep/objects/VolumeManagerInterna.h]: Then you could have just completely removed the line.

**Pull request merged**